draft/2022-02-02-this-week-in-rust.md

```diff
@@ -31,18 +31,22 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 ### Research
 
 ### Observations/Thoughts
+* [The 9 indispensable features to learn for the new Rust programmer](https://kerkour.com/indispensable-rust-features-to-learn/)
 * [Uninitialized Memory: Unsafe Rust is Too Hard](https://lucumr.pocoo.org/2022/1/30/unsafe-rust/)
 * [Part 2: Improving crypto code in Rust using LLVM’s optnone](https://blog.trailofbits.com/2022/02/01/part-2-rusty-crypto/)
 * [Writing the fastest GBDT library in Rust](https://www.tangram.dev/blog/writing_the_fastest_gbdt_library_in_rust/)
 * [Async Rust vs RTOS showdown!](https://tweedegolf.nl/en/blog/65/async-rust-vs-rtos-showdown)
 
 ### Rust Walkthroughs
 
+* [How to implement worker pools in Rust](https://kerkour.com/rust-worker-pool/)
```

Comment:
I'm removing this one because it seems to have been published in July - that's older than we generally feature. Have there been recent changes to it? (If there have, feel free to submit this one for next week's issue)